src/v/kafka/server/group.cc

```diff
@@ -2848,14 +2848,43 @@ ss::future<> group::do_try_abort_old_tx(model::producer_identity pid) {
         }
     }
 
-    std::optional<model::tx_seq> tx_seq;
-
     auto p_it = _prepared_txs.find(pid);
     if (p_it != _prepared_txs.end()) {
-        tx_seq = p_it->second.tx_seq;
-    }
+        auto r = co_await _tx_frontend.local().try_abort(
+          model::partition_id(0), // Partition_id 0???
```

Comment:
since we support serde now can you update `prepare_group_tx_request` to include `model::partition_id` just like we do with rm_stm?

Reply:
We also need to change log record for prepared tx...
I would do it a little bit later, because it is not aurgent right now

Reply:
Add a todo comment with a lint to this issue: https://github.com/redpanda-data/redpanda/issues/6137